changelog.md

```diff
@@ -1,14 +1,14 @@
 ## 9.8.7 / 2018-06-24
 
-* Fix `global` access in the dist version (#736)
```

Comment:
I wonder why we have this all the time, is someones editor using wrong config?

Reply:
these things should have been fixed when you run the linter and prettier over cli, did you get  this changes from editor?

Reply:
Probably, I didn't change that explicitly.